### PR TITLE
Thu/dataset manager tests basics

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: cypress/commons/actions/generic/DatasetManager.js [hunk 1]
```diff
@@ -15,8 +15,81 @@ function switchToDatasetManagerView() {
...
+function getDatasetMetadataSourceType() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.sourceType);
+}
+function getDatasetMetadataApiUrl() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.apiUrl);
+}
+function getDatasetMetadataApiUrlButton() {
+  return cy.get(GENERIC_SELECTORS.datasetmanager.metadata.apiUrlButton);
+}
+function getDatasetMetadataEditDescriptionButton() {
```
Comment (nborde-CSM) [inline cypress/commons/actions/generic/DatasetManager.js:48]: Is there really an edit description button ?

File: cypress/commons/actions/generic/DatasetManager.js [hunk 2]
```diff
@@ -15,8 +15,81 @@ function switchToDatasetManagerView() {
   return getDatasetManagerTab().click();
 }
 
+function getNoDatasetsPlaceholder() {
```
Comment (nborde-CSM) [inline cypress/commons/actions/generic/DatasetManager.js:18]: Should not _dataset_ be in the singular if it has none ?

File: cypress/commons/constants/generic/IdConstants.js [hunk 3]
```diff
@@ -142,8 +142,29 @@ export const GENERIC_SELECTORS = {
...
+      creationDate: '[data-cy=dataset-metadata-creation-date]',
+      refreshDate: '[data-cy=dataset-metadata-refresh-date]',
+      sourceType: '[data-cy=dataset-metadata-source-type]',
+      apiUrl: '[data-cy=dataset-metadata-api-url]',
+      apiUrlButton: '[data-cy=dataset-metadata-copy-api-url-button]',
+      editDescriptionButton: '[data-cy=edit-description]',
+      descriptionTextfield: '[data-cy=description-textfield]',
+      description: '[data-cy=dataset-metadata-description]',
+    },
+    noDatasetsPlaceholder: '[data-cy=no-datasets-placeholder]',
```
Comment (nborde-CSM) [inline cypress/commons/constants/generic/IdConstants.js:158]: Dataset in the singular ?

File: src/views/DatasetManager/components/NoDatasetsPlaceholder/NoDatasetsPlaceholder.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ import { CreateDatasetButton } from '../CreateDatasetButton';
 export const NoDatasetsPlaceholder = () => {
   const { t } = useTranslation();
   return (
-    <Grid container sx={{ height: 1, p: 1 }}>
+    <Grid container sx={{ height: 1, p: 1 }} data-cy="no-datasets-placeholder">
```
Comment (nborde-CSM) [inline src/views/DatasetManager/components/NoDatasetsPlaceholder/NoDatasetsPlaceholder.js:12]: Dataset in the singular ?

**Review by nborde-CSM — COMMENTED** [review]
Two remarks but LGTM :+1:

File: cypress/commons/actions/generic/DatasetManager.js (hunk 2 above)
Comment (csm-thu) [reply cypress/commons/actions/generic/DatasetManager.js:18]: In English, "no", "zero" and "any" are followed by plural nouns :slightly_smiling_face:

File: cypress/commons/actions/generic/DatasetManager.js (hunk 1 above)
Comment (csm-thu) [reply cypress/commons/actions/generic/DatasetManager.js:48]: Yes, it's the pencil icon that appears when hovering the dataset description

**Pull request merged**